new/LIT7279QeneAlbeki.xml

```diff
@@ -0,0 +1,77 @@
+                    <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                        Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                    <pubPlace>Hamburg</pubPlace>
+                    <availability>
+                        <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                            licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                    </availability>
+                </publicationStmt>
+                <sourceDesc>
+                    <p/>
+                </sourceDesc>
+            </fileDesc>
+            <encodingDesc>
+                <p>A digital born TEI file</p>
+            </encodingDesc>
+            <profileDesc>
+                <creation/>
+                <abstract>
+                    <p>Qǝne probably with a reference to <ref type="work" corresp="LIT2362Songof"/>.
+                        Neither type nor author of the qǝne are indicated in the manuscript. According to the classification by 
+                        <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>,
+                        a qǝne of the kʷǝllǝkǝmu-type.</p>
+                </abstract>
+                <textClass>
+                    <keywords>
+                        <term key="ChristianLiterature"/>
+                        <term key="Poetry"/>
+                        <term key="Qene"/>
+                        <term key="Kwellekomu"/>
+                    </keywords>
+                </textClass>
+                <langUsage>
+                    <language ident="en">English</language>
+                    <language ident="gez">Gǝʿǝz</language>
+                </langUsage>
+            </profileDesc>
+            <revisionDesc>
+                <change who="CH" when="2025-02-13">Created entity</change>
+            </revisionDesc>
+        </teiHeader>
+        <text>
+            <body>
+                <div type="edition" xml:lang="gez">
+                    <note>This edition is based on <ref type="mss" corresp="BNFabb145"/>, fol. 56rb-c. Division into verses is indicated with ። , 
+                        except for verse no. 5, where the line verse break was omitted. The end of the poem is indicated with ። ። .</note>
+                    <ab>
+                        <l n="1">አልብኪ፡ ነውር፡ መርዓተ፡ ሊባኖስ፡ ዳእሙ፡ አይሁድ፡ ሰብአ፡ ሀገርኪ፡ ሴሎ።</l> 
+                        <l n="2">ይሁቡኪ፡ ነውረ፡ በዘፈቀዱ፡ ተኃጕሎ።</l>
+                        <l n="3">ደመና፡ ብሩህ፡ መንፈስ፡ ቅዱስ፡ እምአመ፡ ለኬፋ፡ ሥጋኪ፡ ጸለሎ።</l>
+                        <l n="4">እንዘ፡ ኢየአምር፡ አንቀልቅሎ።</l>
+                        <l n="5">ማኅፈደ፡ መለኮት፡ <cb n="56rc"/> ከርሥኪ፡ በትድምርተ፡ ዮሴፍ፡ ዓውሎ<supplied reason="omitted">።</supplied></l> 
+                        <l n="6">ወብዙኃ፡ ይፄዕሉኪ፡ ጽዕለተ፡ ደናግል፡ ኵሎ።</l>
+                        <l n="7">ፅዕዱተ፡ አስናን፡ እንዘ፡ ይብሉ፡ ፈ<choice resp="CH"><sic>ሪ</sic><corr>ረ</corr></choice>ሳውያን፡ በተሀብሎ።</l>
```

Comment:
I think that the reading is ፈሪሳውያን for "Pharisees", cp. ሪ in እግሪሁ in col. 329 (ፈረሳዊ means "horseman").